src/components/UserPortal/CreateGroupChat/CreateGroupChat.tsx

```diff
@@ -94,9 +93,9 @@ export default function CreateGroupChat({
   createGroupChatModalisOpen,
   chatsListRefetch,
 }: InterfaceCreateGroupChatProps): JSX.Element {
-  const { t } = useTranslation('translation', {
-    keyPrefix: 'userChat',
-  });
+  // const { t } = useTranslation('translation', {
+  //   keyPrefix: 'userChat',
+  // });
```

Comment:
This was causing eslint error while commiting changes. Thus, in order to solve esLint i had to comment it because eslint was telling that t is not in use anywhere in code so remove it.

Reply:
Okay, i am doing it.

Reply:
I removed these lines.